include/data_structure/queue.h

```diff
@@ -0,0 +1,85 @@
+/*
```

Comment:
remove.

I have renamed it to `vqueue` in the master.

Why didn't the change reflect when you rebased/merged? 😕